src/Core/src/Handlers/Picker/PickerHandler.iOS.cs

```diff
@@ -60,16 +58,23 @@ protected override MauiPicker CreateNativeView()
 
 		protected override void ConnectHandler(MauiPicker nativeView)
 		{
+			nativeView.EditingDidBegin += OnStarted;
 			nativeView.EditingDidEnd += OnEnded;
 			nativeView.EditingChanged += OnEditing;
+
+			((INotifyCollectionChanged)VirtualView.Items).CollectionChanged += OnRowsCollectionChanged;
```

Comment:
Cast check maybe? The items set by the user aren't going to be always by INCC.

Reply:
Added checks in these cases. Thnx.